src/waveform/renderers/waveformmarkset.cpp

```diff
@@ -70,20 +68,15 @@ void WaveformMarkSet::setup(const QString& group, const QDomNode& node,
             if (controlItemSet.insert(hotCueControlItem).second) {
                 //qDebug() << "WaveformRenderMark::setup - Automatic mark" << hotCueControlItem;
                 WaveformMarkPointer pMark(new WaveformMark(i));
-                WaveformMarkProperties defaultProperties = m_defaultMark.getProperties();
+                WaveformMarkProperties defaultProperties = m_pdefaultMark->getProperties();
                 pMark->setProperties(defaultProperties);
-                pMark->m_pPointCos = std::make_unique<ControlProxy>(pHotcue->getKey());
+                pMark->changeKeyPosition(pHotcue);
```

Comment:
Any results here? I think  we can get rid of this and the WaveformMark::WaveformMark(int hotCue)